### PR TITLE
Created wrapper to hold img ratio same as PromoEventCard

### PR DESCRIPTION
**Pull request by ValkovMirec**
- calculated ration on example card 
- created wrapper arround images where needed to hold that ratio
- done TODO in notice card: Use "MImage" component

### CODE REVIEW

File: next/modules/cards-and-rows/EventCard.tsx [hunk 1]
```diff
@@ -40,12 +41,13 @@ const EventCard = ({
...
-        src={listingImage?.url || coverImage?.url || Placeholder.src}
-        height="200px"
-      />
+      <div className="relative h-[163px] w-[271px] object-cover">
+        <Image
+          fill
+          alt={t('eventDetailImagePlaceholder')}
+          src={listingImage?.url || coverImage?.url || Placeholder.src}
+        />
+      </div>
```
Comment (radoslavzeman) [inline next/modules/cards-and-rows/EventCard.tsx:50]: Never use fixed width for cards, they must be responsive.

`object-cover` is usually used on Image together with `fill`:
```
<Image fill className="object-cover" .../>
```

Try to use tailwind's `aspect-ratio` class.

File: next/modules/cards-and-rows/NoticeCard.tsx [hunk 2]
```diff
@@ -26,12 +26,11 @@ const NoticeCard = ({ notice }: NoticeCardProps) => {
...
-        <img
-          className="mb-4 h-48 w-full object-cover"
-          alt=""
-          src={image?.attributes?.url || Placeholder.src}
-        />
+        {image?.attributes && (
+          <div className="relative h-[163px] w-[271px] object-cover">
+            <MImage image={image?.attributes} fill />
+          </div>
+        )}
```
Comment (radoslavzeman) [inline next/modules/cards-and-rows/NoticeCard.tsx:33]: This will not show Placeholder if no image is provided...

See also comments above.

**Review by radoslavzeman — CHANGES_REQUESTED** [review]
Yes, image probably needs a wrapping div, but your implementation is not responsive and removes placeholder for NoticeCard.

You can use MImage in both cases, it's even better.

**Comment by daksas303** [comment]
- Removed fixed width
- Added `fallbackImage` into next image `onError` when failed to load image src into `MImage`. Note: There appears to be an issue when having opened Chrome dev tools, that the images randomly flicker (on load, on scroll, etc.). With Chrome dev tools closed it behaves normally, but in case this appears as an issue we will have to figure out a different solution.
- Added object-cover
- Tried the tailwinds aspect ration but it didn't seem to make a difference.

**Pull request closed without merging**